packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable i18next/no-literal-string */
+import { Component, Host, h, Prop, Event, EventEmitter, Fragment } from '@stencil/core';
+
+/**
+ * @componentName Privacy agreement
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+@Component({
+  tag: 'va-privacy-agreement',
+  styleUrl: 'va-privacy-agreement.css',
+  shadow: true,
+})
+export class VaPrivacyAgreement {
+  /**
+   * Whether the checkbox is checked or not.
+   */
+  @Prop({ mutable: true }) checked?: boolean = false;
+
+  /**
+   * Whether to display the error message or not.
+   */
+  @Prop() isError?: boolean = false;
+
+  /**
+   * Emit component-library-analytics events on the blur event.
+   */
+  @Prop() enableAnalytics?: boolean = false;
+
+  /**
+   * The event used to track usage of the component. This is emitted when the
+   * checked value changes and enableAnalytics is true.
+   */
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  /**
+   * The event emitted when the checkbox changes.
+   */
+  @Event() vaChange: EventEmitter;
+
+  private handleCheckboxChange = (e: Event) => {
+    this.checked = (e.target as HTMLInputElement).checked;
+    this.vaChange.emit({ checked: this.checked });
+
+    if (this.enableAnalytics) {
+      this.componentLibraryAnalytics.emit({
+        componentName: 'va-privacy-agreement',
+        action: 'click',
+        details: {
+          checked: this.checked
+        },
+      });
+    }
+  }
+
+  render() {
+    return (
+      <Host>
+        <span>
+          Please read and accept the&nbsp;
+          <a target="_blank" href="/privacy/">
```

Comment:
Is this the link we want to use? https://va.gov/privacy redirects to the [OPRM subdomain](https://www.oprm.va.gov/).

![Screenshot of oprm.va.gov](https://user-images.githubusercontent.com/2008881/189423771-f802f713-433e-4733-91e5-e50b54da19cc.png)

Reply:
Interesting. I think the correct link right now is https://www.va.gov/privacy-policy/ but this was taken directly from the [React version](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/react-components/src/components/PrivacyAgreement/PrivacyAgreement.jsx#L21) of the component so I guess that link changed at some point.

@k80bowman This could be related to what we were talking about before with making the link a prop (defaulting to `/privacy-policy`) instead of hard-coded. If we still want to keep it on lockdown, I can just update this to the current working privacy link but I'm wondering if the old version of this component is still being used somewhere with an incorrect hard-coded link.

Reply:
The React version of this component isn't being used anywhere. [This issue](https://github.com/department-of-veterans-affairs/component-library/issues/333) hasn't been updated in a while, but the four components under "Unused" are still unused.

Reply:
Gotcha. I'll just make sure the link is updated to `/privacy-policy` then. 👍

Reply:
Yep, this is the link to use. Thank you for updating it.